GameTest.java

```diff
@@ -0,0 +1,215 @@
+import static org.junit.Assert.*;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class GameTest {
+
+	Game game;
+	
+	@Before
+	public void setup(){
+		game = new Game();
+	}
+	
+	// In each test which checked the moving function, I asserted both changing places
+	
+	// - Pacman -
+	// Right
+	
+	@Test
+	public void PacmanRightWall(){
+		// arrange
+		this.game.board = new Board(new String[]{"PW"}).getBoard();
```

Comment:
Don't use (set or get) internal members of a class directly. 
A better approach is: 
`this.game = new Game(new Board(new String[]{"PW"}));`
This way you can pass an initial board to the game, but the game itself is "responsible" to utilize the board.